src/pages/account-page/account-page.tsx

```diff
@@ -1,225 +1,32 @@
-
-type TabPanelProps = {
-  children?: ReactNode
-  index: number
-  value: number
-}
-
-type CustomAccordionProps = {
-  index: number
-  expanded: number | null
-  onChange: (newIndex: number | null) => void
-  summary: string
-  children: ReactNode
-}
-
-const CustomTabPanel: FC<TabPanelProps> = ({
-  children,
-  value,
-  index,
-  ...other
-}) => {
-  return (
-    <Box
-      role="tabpanel"
-      hidden={value !== index}
-      id={`simple-tabpanel-${index}`}
-      aria-labelledby={`simple-tab-${index}`}
-      {...other}
-    >
-      {value === index && <Box>{children}</Box>}
-    </Box>
-  )
-}
-
-const a11yProps = (index: number) => ({
-  'id': `simple-tab-${index}`,
-  'aria-controls': `simple-tabpanel-${index}`
-})
-
-const accordions = [
-  {
-    summary: 'Контактна інформація',
-    content: <ContactInfo />
-  },
-  {
-    summary: 'Ваші адреси доставки',
-    content: <DeliveryAddress />
-  },
-  {
-    summary: 'Історія замовлень',
-    content: <OrderHistory />
-  },
-  {
-    summary: 'Змінити пароль',
-    content: <ChangePassword />
-  }
-]
-
-const CustomAccordion: FC<CustomAccordionProps> = ({
-  index,
-  expanded,
-  onChange,
-  summary,
-  children
-}) => (
-  <Accordion
-    expanded={expanded === index}
-    onChange={(_, isExpanded) => onChange(isExpanded ? index : null)}
-    sx={{
-      '&.MuiPaper-root.MuiAccordion-root': {
-        'boxShadow': 'none',
-        '&::before': {
-          display: 'none'
-        }
-      }
-    }}
-  >
-    <AccordionSummary
-      expandIcon={<KeyboardArrowDownIcon sx={{ color: 'secondary.darker' }} />}
-      {...a11yProps(index)}
-      sx={{
-        '.MuiTypography-root': {
-          fontFamily: 'Nunito',
-          fontWeight: 700,
-          fontSize: 17,
-          color: 'secondary.darker'
-        }
-      }}
-    >
-      <Typography>{summary}</Typography>
-    </AccordionSummary>
-    <AccordionDetails sx={{ p: 0 }}>{children}</AccordionDetails>
-  </Accordion>
-)
+import { AccordionsSmallScreen } from '../../components/Account-panel/Account-nav/Accordions'
+import { TabsBigScreen } from '../../components/Account-panel/Account-nav/Tabs'
 
 export const AccountPage = () => {
-  const { user } = useAuth()
-  const theme = useTheme()
-  const [activeIndex, setActiveIndex] = useState<number>(0) // Unified state for tabs and accordions
-  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'))
+  const [searchParams, setSearchParams] = useSearchParams()
   const [openModal, setOpenModal] = useState(false)
 
-  const handleChange = (_event: SyntheticEvent, newValue: number) => {
-    setActiveIndex(newValue)
-  }
+  const initialTabIndex = searchParams.get('tab')
+    ? parseInt(searchParams.get('tab') || '1', 10) - 1
+    : 0
 
-  const handleAccordionChange = (newIndex: number | null) => {
-    setActiveIndex(newIndex ?? 0)
-  }
+  const [activeIndex, setActiveIndex] = useState(initialTabIndex)
```

Comment:
why do you need this state if you alreade have index in `url`?

Reply:
remove